sdk/identity/azure-identity/tests/test_cae.py

```diff
@@ -17,27 +15,12 @@
 )
 from azure.identity._constants import DEVELOPER_SIGN_ON_CLIENT_ID
 from azure.mgmt.resource.subscriptions import SubscriptionClient
-from azure_devtools.scenario_tests import GeneralNameReplacer, RequestUrlNormalizer, patch_time_sleep_api
-from devtools_testutils import AzureTestCase
+from devtools_testutils import AzureRecordedTestCase, is_live, recorded_by_proxy
 import pytest
-import requests
-from six.moves.urllib_parse import urlparse
 
-from recording_processors import IdTokenProcessor, RecordingRedactor
-
-
-@pytest.mark.skip("these tests require support in azure-core")
-class CaeTestCase(AzureTestCase):
-    def __init__(self, *args, **kwargs):
-        scrubber = GeneralNameReplacer()
-        super(CaeTestCase, self).__init__(
-            *args,
-            recording_processors=[RecordingRedactor(record_unique_values=True), scrubber],
-            replay_processors=[RequestUrlNormalizer(), IdTokenProcessor()],
-            **kwargs
-        )
-        self.scrubber = scrubber
-        if self.is_live:
+class CaePreparer(object):
```

Comment:
Is the idea to use this once we can run these tests (when there's support in `azure-core`)?

Reply:
Yes.